src/finn/transformation/fpgadataflow/set_folding.py

```diff
@@ -120,7 +120,7 @@ def apply(self, model):
         ]
         # these ops are preceded by depthwise SWG and have special behavior,
         # as explained in the SetFolding docstring
-        depthwise_op_exceptions = ["VectorVectorActivation_hls", "Pool_hls"]
+        depthwise_op_exceptions = ["VVAU_hls", "Pool_hls"]
```

Comment:
Please add VVAU_rtl and check if there are other transformations in which this should be included.